packages/web/src/components/earn-add/earn-add-confirm/EarnAddConfirm.tsx

```diff
@@ -67,7 +68,8 @@ const EarnAddConfirm: React.FC<EarnAddConfirmProps> = ({
 
       <Button
         text="Confirm Add Position"
-        onClick={confirm}
+        onClick={feeInfo.errorMsg ? undefined : confirm}
+        disabled={!!feeInfo.errorMsg}
```

Comment:
Define the onClick function.
It doesn't look good to intentionally use an undefined value.